src/ngraph/ngraph_sgcompiler.cc

```diff
@@ -36,17 +35,7 @@
 
 namespace ngraph_bridge {
 
-static int fcount = 0;
-
-void dump_graph(std::shared_ptr<ngraph::Function> f) {
-  std::stringstream fname;
-  fname << "Graph_" << fcount << ".json";
-  fcount += 1;
-  std::ofstream file;
-  file.open(fname.str());
-  file << ngraph::serialize(f) << std::endl;
-  file.close();
-}
+NgGraphDumper gdumper;
```

Comment:
I think this should be static?

Reply:
Oops.  I got that right the first time I wrote the code.  But I forgot that detail then I re-applied the code changes because of the repo branch renames last night :) . Will fix.